corehq/motech/generic_inbound/views.py

```diff
@@ -126,5 +133,23 @@ def post(self, request, domain, **kwargs):
         return self.get(request, domain, **kwargs)
 
 
+@json_error
+@api_auth
+@api_throttle
+@require_http_methods(["POST"])
 def generic_inbound_api(request, domain, api_id):
-    pass
+    try:
+        api = ConfigurableAPI.objects.get(key=api_id, domain=domain)
```

Comment:
I don't think this PR is necessarily the place for it, but we should think about how to ensure this transformation is efficient - if an automated process is bombarding us with case updates, it'd be nice to have the transformation function maybe even in memory.  Though if it's a get-by-id maybe it's comparable to all the other overhead anyways, I dunno.

Reply:
I did think about trying to cache it but it seemed premature. Also that means having to deal with cache invalidation.